modules/icinga/files/usr/lib/nagios/plugins/check_json_healthcheck

```diff
@@ -151,7 +158,7 @@ if __name__ == "__main__":
         with closing(urlopen(json_request(check_url), timeout=20)) as response:
             healthcheck_info = HealthCheckInfo(json.load(response))
     except HTTPError as e:
-        report_error("healthcheck returned HTTP error %d" % (e.code,))
+        report_error("healthcheck returned HTTP/HTTPS error %d" % (e.code,))
```

Comment:
HTTPS doesn't have specific error codes, it's just HTTP over TLS. So HTTP error is correct in either case.